client/cody/src/chat/protocol.ts

```diff
@@ -45,14 +45,36 @@ export type ExtensionMessage =
  */
 export interface ConfigurationSubsetForWebview extends Pick<Configuration, 'debugEnable' | 'serverEndpoint'> {}
 
+/**
+ * URLs for the Sourcegraph instance and app.
+ */
 export const DOTCOM_URL = new URL('https://sourcegraph.com')
+export const DOTCOM_CALLBACK_URL = new URL('https://sourcegraph.com/user/settings/tokens/new/callback')
+export const CODY_DOC_URL = new URL('https://docs.sourcegraph.com/cody')
+// Community and support
+export const DISCORD_URL = new URL('https://discord.gg/s2qDtYGnAE')
+export const CODY_FEEDBACK_URL = new URL(
+    'https://github.com/sourcegraph/sourcegraph/discussions/new?category=product-feedback&labels=cody,cody/vscode'
+)
+// APP
 export const LOCAL_APP_URL = new URL('http://localhost:3080')
+export const APP_LANDING_URL = new URL('https://about.sourcegraph.com/app')
+export const APP_CALLBACK_URL = new URL('sourcegraph://user/settings/tokens/new/callback?requestFrom=CODY')
+export const APP_DOWNLOAD_URLS: { [os: string]: { [arch: string]: string } } = {
+    darwin: {
+        arm64: 'https://github.com/sourcegraph/sourcegraph/releases/download/app-v2023.6.13%2B1311.1af08ae774/Sourcegraph_2023.6.13+1311.1af08ae774_aarch64.dmg',
+        x64: 'https://github.com/sourcegraph/sourcegraph/releases/download/app-v2023.6.13%2B1311.1af08ae774/Sourcegraph_2023.6.13+1311.1af08ae774_x64.dmg',
```

Comment:
```suggestion
        x64: 'https://github.com/sourcegraph/sourcegraph/releases/download/app-v2023.6.16%2B1314.6c2d49d47c/Cody_2023.6.16+1314.6c2d49d47c_x64.dmg',
```
These are the latest download links and the first for the "Cody App" the previous version we likely will not autoupdate.  These will get out date quickly but at least the app should auto update from this version on.